lib/matplotlib/axes/_axes.py

```diff
@@ -1975,6 +1975,8 @@ def bar(self, left, height, width=0.8, bottom=None, **kwargs):
         if not self._hold:
             self.cla()
         color = kwargs.pop('color', None)
+        if color is None:
```

Comment:
Done but used a different set of aliases.  Passing in `c` currently does not work, no reason to let it start working.